kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -413,33 +413,39 @@ public int getInitialPartitionCount()
     return initialPartitionCount;
   }
 
-  public void setClusters(String clusters)
-  {
-    this.clusters = clusters.split(";");
-  }
-
   /**
    *  Same setting as bootstrap.servers property to KafkaConsumer
    *  refer to http://kafka.apache.org/documentation.html#newconsumerconfigs
-   *  To support multi cluster, you can have multiple bootstrap.servers separated by ";"
+   *  To support multi cluster, you can have multiple elements in the array
    */
-  public String getClusters()
+  public void setClusters(String[] clusters)
   {
-    return Joiner.on(';').join(clusters);
+    this.clusters = clusters;
```

Comment:
The clusters are separated by commas. In case the clusters themselves have multiple host:port pairs, the string can be specified using quotes as follows: "host1:port1, host2:port2", "host3:port3".